packages/lexical-table/src/LexicalTableCellNode.ts

```diff
@@ -49,6 +49,7 @@ export type SerializedTableCellNode = Spread<
     headerState: TableCellHeaderState;
     width?: number;
     backgroundColor?: null | string;
+    verticalAlign?: null | string;
```

Comment:
No reason to allow null to be serialized here 

```suggestion
    verticalAlign?: string;
```